src/server/api/schema.js

```diff
@@ -3810,6 +3830,14 @@ const rootResolvers = {
         prevCampaignId,
         nextCampaignId
       };
+    },
+    superadmins: async (_root, _options, { user }) => {
+      if (user.is_superadmin) {
```

Comment:
Question: Should this be available to supervols and higher instead? Ex. On the regular People page, all users with access can view who's a superadmin. The People page of course doesn't currently refer to this field, but could imagine future references to it which are similar to that.

Reply:
I think if we need this for a specific use case in the future, we can evaluate it then, currently superadmins are only accessible to other superadmins, and themselves. (below comment). I don't see a specific reason to change this behavior, and would prefer to keep permissions closed down as default, rather than opening up for a future possibility. 

@bchrobot thoughts?

Reply:
Sounds good 👍🏾